commands/user.go

```diff
@@ -402,3 +427,84 @@ func listUsersCmdF(c client.Client, command *cobra.Command, args []string) error
 
 	return nil
 }
+
+func generateTokenForAUserCmdF(c client.Client, command *cobra.Command, args []string) error {
+	if len(args) != 2 {
+		return errors.New("expected at least two arguments. See help text for details")
+	}
```

Comment:
If we apply the `ExactArgs` suggestion we can safely remove this check

Reply:
Added. 👍